### PR TITLE
fix: add a default column name title of Result

### LINKED ISSUE #1662: FEAT: Sensible default for blank or whitespace only result column titles
**Is your feature request related to a problem? Please describe.**
When running a query such as
```sql
select count(*) from sys.procedures
```

The results column header shows up as `&nbsp;` unless I also alias the result like in the following

```sql
select count(*) as stored_procedures from sys.procedures
```

**Describe the solution you'd like**
I'd like unaliased results for unnamed results to be given a default name such as "result count", "results" or similar instead of the escaped string value.

**Describe alternatives you've considered**
While always aliasing results is probably best practice, it would be nice to have a sensible default.

**Additional context**
Screenshot 2023-08-10 at 9 22 15 AM

**Database type, if appropriate**
This affects all databases.

### PR DESCRIPTION
**Pull request by wmontgomery**
fix #1662

record-count

### CODE REVIEW

File: apps/studio/src/components/editor/ResultTable.vue [hunk 1]
```diff
@@ -179,8 +179,9 @@
         const columnWidth = this.result.fields.length > 30 ? globals.bigTableColumnWidth : undefined
         return this.result.fields.map((column) => {
           console.log('COLUMN: ', column);
+          console.log(column.name)
```
Comment (not-night-but) [inline apps/studio/src/components/editor/ResultTable.vue:182]: I'd just get rid of the logs here

**Review by not-night-but — APPROVED** [review]

**Review by rathboma — CHANGES_REQUESTED** [review]
just kill the logs and we're good to go!

File: apps/studio/src/components/editor/ResultTable.vue (hunk 1 above)
Comment (wmontgomery) [reply apps/studio/src/components/editor/ResultTable.vue:182]: Oops. Consider them gone!

**Comment by wmontgomery** [comment]
> what if there are multiple unnamed columns? Are they all `Result`?

Yeah there would be. It's not a frequent case (at least I hope not as you should be aliasing your columns). I can add some kind of counter to it if wanted. That's actually fairly easy since it's a map.

Something like this just to get the index in there
```js
return this.result.fields.map((column, index) => {
  const result = {
    title: column.name || `Result ${index}`,
```

**Review by rathboma — APPROVED** [review]

**Pull request merged**